### PR TITLE
add basic structure of non US banner for feb moment for non-supporters

### PR DESCRIPTION
**Pull request by johnduffell**
## What does this change?
This adds the main structure of the banner for the feb moment.
This is not fully styled but it makes sense to get the logic out first.
It is behind a switch so it won't go out until we switch it on, we will do more PRs before then (tues)
This test is for the existing supporters outside the US.  There will be alternatives for US, and for people with existing supporter relationships.

We are waiting for the proper SVG for the main headline, we have work to do on the circles to get the black colour where the overlap, and we need the "read more" link.  There are also a lot of obvious copy/styling tweaks needed.
## Screenshots
image

## What is the value of this and can you measure success?
This is to support the moment, which we hope will be successful in terms of getting annualised value (as well as recognition from readers about what they are contributing towards)

### Tested

- YES Locally
- NO On CODE (optional)

@joelochlann @jranks123 @ionamckendrick

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    'essential editorial independence. This approach ' +
+                    'allows us to keep our journalism accessible to all, ' +
+                    'so more people, across the world, have access ' +
+                    'to accurate information with integrity at its heart.',
+                // buttonCaption?: string, TO be decided with non engineers
+                // linkUrl?: string,
+                // hasTicker?: boolean,
+                // products?: OphanProduct[],
+                template: acquisitionsBannerFivTemplate,
+                // bannerModifierClass?: string,
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js]: you probably want the same `bannerModifierClass` here too

File: static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                // minArticlesBeforeShowingBanner?: number,
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js]: you need `minArticlesBeforeShowingBanner: 0` for both variants

Comment (joelochlann) [reply static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner-non-us.js]: and it's probably worth getting rid of the commented ones now, you can always check the type to find out other ones

**Review by joelochlann — APPROVED** [review]
Looks good, let's just get those `minArticlesBeforeShowingBanner` and `bannerModifierClass` params on before we forget!

**Comment by johnduffell** [comment]
@joelochlann thanks for the comments I will do them straight away in the next PR (to avoid forgetting) I have merged early so Tom and I can pair and Jonny can do more styling based off this

**Comment by johnduffell** [comment]
I have checked in prod and the old normal banner is still showing! :)

**Pull request merged**